### DISCUSSION TITLE
Opinions about excaliburjs

### DISCUSSION THREAD

**Discussion by kgish** [excaliburjs/Excalibur#2321] [General]
Hi there. I've been searching for a good javascript game engine based on typescript, and so far excaliburjs seems to be one of the few half decent tools available.

However, one thing that worries me is the low version and whether or not excaliburjs will be sustained and is future proof.

Would be curious to hear from other users about this, including opinions about support and the planned roadmap.

**Comment by eonarheim** [comment]
Hi @kgish 

Thanks for the kind words! We've been working on Excalibur for nearly 10 years!

As the lead maintainer I'll throw my 2 cents in:
* I'd love to hear any ideas, features, or suggestions for the roadmap as proposal issues!
* We are pre-1.0 still because we may change some APIs and there will be some breaking changes between minor versions (v0.25.x -> v0.26.0 for example), however I expect these to be relatively small/isolated at this point.
* [1.0 is coming soon](https://github.com/excaliburjs/Excalibur/issues/1161) (the next year or two) there isn't too much more we have planned in v0.27.0 & v0.28.0 at the moment

**Comment by twsau** [comment]
Just stumbled across the repo. Very interesting stuff! I've had a lot of fun with libraries like pixi.js in the past. Looking forward to playing around with excaliburjs. 👍
